app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+  end
+
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
It will be great to create a Like on behalf of the currently logged-in user. At the moment in this assignment, we are assuming the currently logged-in user is the one that is defined in your application controller as the current_user method. So when you are creating your like, don't get the user_id from your params get the user id from the @current_user.